commands/list_test.go

```diff
@@ -0,0 +1,72 @@
+// pmm-admin
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package commands
+
+import "testing"
+
+func TestListResultString(t *testing.T) {
+	type fields struct {
+		Services []listResultService
+		Agents   []listResultAgent
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   string
+	}{
+		{
+			name: "filled",
+			fields: fields{
+				Services: []listResultService{
+					{ServiceType: "mysql", ServiceID: "/service_id/test", ServiceName: "mysql-service"},
```

Comment:
Let's use real-looking IDs. One of the things we want to test is columns formatting/alignment.